week-1/K-functions-parameters/exercise4.js

```diff
@@ -1,5 +1,8 @@
 // Declare your function first
-
+function adding(a, b) {
```

Comment:
This looks good, but how could you name the function a bit better to make it clearer. Normally we name functions as verbs, e.g. `create`, `jump`, `doTask`